src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameHelpers.StrongName.cs

```diff
@@ -7,7 +7,12 @@
 
 namespace System.Reflection
 {
-    public static partial class AssemblyNameHelpers
+#if CORERT
+    public // Needs to be public so that Reflection.Core can see it.
```

Comment:
It would be better to expose the method that Reflection.Core needs using `Internal.Runtime.Augments.RuntimeAugments` and keep this internal, without ifdefs.